src/utils/dateHelpers.ts

```diff
@@ -0,0 +1,4 @@
+import moment from 'moment';
+
+export const getNextMonday = () =>
+  moment().startOf('week').add(1, 'week').day('monday').format('MMMM Do');
```

Comment:
```suggestion
  moment().utc().startOf('week').add(1, 'week').day('monday').format('MMMM Do');
```

otherwise it will use the local time and wont switch synchronized for everybody.